pegasus/src/homepage.rb

```diff
@@ -506,15 +506,15 @@ def self.show_curriculum_banner(request)
   end
 
   def self.show_professional_learning_banner(request)
-    false
+    request.locale == "en-US"
   end
 
   def self.show_courses_banner(request)
     false
   end
 
   def self.show_special2020_banner(request)
```

Comment:
Should we be keeping this around now that's it's no longer 2020?

Reply:
that's a great question. @clareconstantine?